docs/arlas-security.md

```diff
@@ -0,0 +1,90 @@
+# ARLAS security
+
+This page describes how to configure ARLAS in order to control access to resources.
+
+## Authentication
+Authentication is the process of identifying a user and is a prerequisite to [Authorisation](#authorisation).  
+Depending on the use case, some configuration must be set. Refer to [ARLAS server authentication configuration](arlas-server-configuration.md)
+
+* Use case 1: public access, no endpoint is protected (e.g. dev, test deployment)
+  - ARLAS server:  set `arlas_auth.enabled` to `false`
+  - ARLAS wui: do not set json property `arlas.authentication` in `config.json` (TODO: add link to WUI or add doc)
+
+* Use case 2: public access, some endpoints are protected (e.g. demo, freemium deployment)
+  - ARLAS server:  set `arlas_auth.enabled` to `true`
+  - ARLAS server:  set `arlas_auth.public_uris` to the needed value, 
+  e.g. `swagger.*:*,explore/.*:*` will only allow public access to URIs `/swagger.*` and 
+  `/explore/.*`
+  - ARLAS wui: do not set json property `arlas.authentication` in `config.json` (TODO: add link to WUI or add doc)
+
+* Use case 3: protected access (e.g. customer deployment)
```

Comment:
you should mention the fact that the protection here **does** impact the WUI and that therefore no some configuration is required in the WUI for security.

Reply:
done